server/utils/utils.ts

```diff
@@ -109,7 +109,7 @@ const stringToAddressObject = (address: string): Address => {
 
 const jsonDtToDate = (dt: string): string => {
   const momentDate = moment(dt, 'DD/MM/YYYY HH:mm')
-  return momentDate.isValid() ? momentDate.format('Do MMMM YYYY') : null
+  return momentDate.isValid() ? momentDate.format('D MMMM YYYY') : null
```

Comment:
Design confirmed we should be displaying cardinal dates (eg 1 December 2024) rather than ordinal dates (eg 1st December 2024) across the service so altered the date formatting here.